featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -637,9 +637,9 @@ def parallel_calculate_chunks(cutoff_time, chunk_size, feature_set, approximate,
         chunks = [df for _, df in chunks]
 
         if len(chunks) < num_workers:
```

Comment:
I think the same block-level pragma could go here and save some line changes.

Reply:
Fixed:
https://github.com/alteryx/featuretools/pull/1829/commits/a1ac50e2b4fdb5c767f474154d97e198c66e0294